pool/pool.go

```diff
@@ -1788,40 +1808,10 @@ func (p *Pool) Close() {
 	<-p.closedCh
 }
 
-// creates new session token with specified owner from SessionCreate call result.
-func sessionTokenForOwner(id user.ID, cliRes *resCreateSession, exp uint64) *session.Token {
-	st := session.NewToken()
-	st.SetOwnerID(&id)
-	st.SetID(cliRes.id)
-	st.SetSessionKey(cliRes.sessionKey)
-	st.SetExp(exp)
-
-	return st
-}
-
 func newAddressFromCnrID(cnrID *cid.ID) *address.Address {
 	addr := address.NewAddress()
 	if cnrID != nil {
 		addr.SetContainerID(*cnrID)
 	}
 	return addr
 }
-
-func copySessionTokenWithoutSignatureAndContext(from session.Token) (to session.Token) {
```

Comment:
Are we sure this function is unnecessary ?

Reply:
Previously we used structure with `interface{}` which was pointer to the context, therefore we could modify it. With separate structure like `session.Object` the issue is gone, I think. At least cache test with context modification is passed.